package.json

```diff
@@ -111,6 +110,10 @@
     "webpack-bundle-analyzer": "^4.7.0",
     "webpack-cli": "^5.0.0"
   },
+  "engines": {
+    "npm": ">= 6.0.0",
```

Comment:
we don't technically need to define npm here since we're not actually using npm.

Reply:
well.. this is just copied from demosplan-core. however, 6b3e9e39f1c4c96a645b827a082ff3210004b20e